lib/resilient/circuit_breaker.rb

```diff
@@ -1,9 +1,27 @@
 require "resilient/key"
 require "resilient/circuit_breaker/metrics"
 require "resilient/circuit_breaker/properties"
+require "resilient/circuit_breaker/registry"
 
 module Resilient
   class CircuitBreaker
+    # Public: Resets all registered circuit breakers (and thus their state/metrics).
+    # Useful for ensuring a clean environment between tests.
+    def self.reset
```

Comment:
I don't want people to use Registry.default.reset. I'd prefer that to stay internal so I'm leaving this method.